runner/entrypoint.sh

```diff
@@ -1,5 +1,12 @@
 #!/bin/bash
 
+if [ -z "${GITHUB_URL}" ]; then
+  echo "Working with public GitHub" 1>&2
+  GITHUB_URL="https://github.com"
+else
+  echo "Github endpoint URL ${GITHUB_URL}"
```

Comment:
Would you mind adding additional validation in the controller that ensures GITHUB_URL envvar to have the trailing slash?

Reply:
There needs to be some more logic added too, as client's baseURL is by default api.github.com, not github.com.